Dockerfile

```diff
@@ -17,8 +17,6 @@ ADD . .
 
 RUN go get -v -t $(go list ./... | grep -v /vendor/)
 
-RUN go install -v ./buffalo
-
```

Comment:
> Definitely needed. It's compiling the local source.

OK, but I've tested by adding a panic to new generator and the `buffalo new` commands started to panic. The go get should still install the local version.

But I'll add it back now.